### PR TITLE
docs: update grammar in readme

### PR DESCRIPTION
**Pull request by aditya20t**

### CODE REVIEW

**Comment by RandellDawson** [comment]
@raisedadead I think most of the changes are good.  The only change I question is changing "Platform" to "The platform".

**Comment by adad20** [comment]
> @raisedadead I think most of the changes are good. The only change I question is changing "Platform" to "The platform".

Thank you for your suggestion, I have updated the readme.md file again.

File: README.md [hunk 1]
```diff
@@ -123,15 +123,20 @@ Our community also has:
 - A [forum](https://www.freecodecamp.org/forum) where you can usually get programming help or project feedback within hours.
 - A [YouTube channel](https://youtube.com/freecodecamp) with free courses on Python, SQL, Android, and a wide variety of other technologies.
 - A [podcast](https://podcast.freecodecamp.org/) with technology insights and inspiring stories from developers.
+
+- A comprehensive [guide to thousands of programming topics](https://guide.freecodecamp.org/)
+- A [Developer News](https://www.freecodecamp.org/news) publication, a free, open-source, no-ads place to cross-post your blog articles.
+=======
```
Comment (raisedadead) [inline README.md]: You don't need these lines

File: README.md [hunk 2]
```diff
@@ -123,15 +123,20 @@ Our community also has:
...
 - A [Developer News](https://www.freecodecamp.org/news) publication, a free, open source, no-ads place to cross-post your blog articles.
 
+
```
Comment (raisedadead) [inline README.md]: You don't need this empty new line

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by adad20** [comment]
All the issues has been resolved.

**Review by raisedadead — CHANGES_REQUESTED** [review]
> All the issues has been resolved.

You just simple clicked the resolve button?

image


That will not make the code change for you! Please make changes, as requested before.

**Comment by adad20** [comment]
I am sorry I have updated it on my local repository. Now I have resolved all the issues.

File: README.md [hunk 3]
```diff
@@ -125,13 +125,13 @@ Our community also has:
 - A [podcast](https://podcast.freecodecamp.org/) with technology insights and inspiring stories from developers.
 - A [Developer News](https://www.freecodecamp.org/news) publication, a free, open source, no-ads place to cross-post your blog articles.
 
+
```
Comment (RandellDawson) [inline README.md]: Please remove this extra line:
```suggestion
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Please fix the one change @raisedadead requested earlier.

**Review by RandellDawson — APPROVED** [review]
@raisedadead Feel free to merge if you are fine with the "The platform" change.

File: README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Our full-stack web development curriculum is completely free and self-paced. We
 * [Reporting Bugs and Issues](#reporting-bugs-and-issues)
 * [Reporting Security Issues](#reporting-security-issues)
 * [Contributing](#contributing)
-* [Platform, Build and Deployment Status](#platform-build-and-deployment-status)
+* [The platform, Build and Deployment Status](#platform-build-and-deployment-status)
```
Comment (raisedadead) [inline README.md]:
```suggestion
* [Platform, Build and Deployment Status](#platform-build-and-deployment-status)
```

This should be understood as platform status, build status and deployment status as three things.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**